### PR TITLE
Edit setup.py, README and import statements

### PR DESCRIPTION
**Pull request by shravandoda**
@Mec-iS kindly review

### CODE REVIEW

File: hydra_python_core/doc_maker.py [hunk 1]
```diff
@@ -2,9 +2,9 @@
 create a HydraDoc object for it'''
 import re
 import json
-from hydrus.samples.doc_writer_sample import api_doc as sample_document
-from hydrus.hydraspec.doc_writer import HydraDoc, HydraClass, HydraClassProp, HydraClassOp
-from hydrus.hydraspec.doc_writer import HydraStatus
+from samples.doc_writer_sample import api_doc as sample_document
```
Comment (xadahiya) [inline hydra_python_core/doc_maker.py:5]: Don't think this will work as samples is not a python module

Comment (xadahiya) [inline hydra_python_core/doc_maker.py:5]: Don't think this will work as samples is not a python module

Comment (Mec-iS) [reply hydra_python_core/doc_maker.py:5]: @ShravanDoda why you would ever want to import a sample in one of the modules? Samples are there as examples, not to be used in the code.

Comment (shravandoda) [reply hydra_python_core/doc_maker.py:5]: @xadahiya @Mec-iS 
`doc_maker` uses `doc_writer_sample` when it's being run directly and not being imported.
<pre>if __name__ == "__main__":
    api_doc = create_doc(sample_document.generate())
    print(json.dumps(api_doc.generate(), indent=4, sort_keys=True))</pre>

Comment (shravandoda) [reply hydra_python_core/doc_maker.py:5]: This is the way it's being done in `hydraspec`

Comment (Mec-iS) [reply hydra_python_core/doc_maker.py:5]: then `doc_maker` should be implemented as library module only and that `__main__` block removed.
If we want to keep that in the package, it should be moved into a `main.py` or `cli.py` module.

Comment (shravandoda) [reply hydra_python_core/doc_maker.py:5]: @xadahiya
`doc_writer_sample` is a python module. It uses `doc_writer` to create a `doc_writer_sample_output.py`.

Comment (shravandoda) [reply hydra_python_core/doc_maker.py:5]: @Mec-iS shall I add a commit for removing the `__main__` block?

**Comment by Mec-iS** [comment]
Fixed and moved to #13

File: hydra_python_core/doc_maker.py (hunk 1 above)
Comment (Mec-iS) [reply hydra_python_core/doc_maker.py:5]: this PR has been closed and moved. See comment below.

**Pull request closed without merging**